lib/ramble/ramble/cmd/workspace.py

```diff
@@ -51,6 +51,7 @@
     'concretize',
     'setup',
     'analyze',
+    'pushtocache',
```

Comment:
Can names have hyphens?

Reply:
Right now, in short, no 

The reason is that we directly map this dict to a function named `workspace_{key}` which would mean we need a function with hypens in. Not impossible to remap if we do want it but does not work out of the box today

Reply:
Updated this functionality in a seperate PR (#286) and added it here